config/environments/development.rb

```diff
@@ -36,7 +36,7 @@
   config.action_mailer.perform_caching = false
   config.action_mailer.delivery_method = :notify
   config.action_mailer.notify_settings = {
-    api_key: ENV.fetch('GOVUK_NOTIFY_API_KEY')
+    api_key: ENV['GOVUK_NOTIFY_API_KEY']
```

Comment:
Why this change? `fetch` would throw an exception if the key weren't set

Reply:
See my comment to @tijmenb above. He raised the same query.